packages/tools/client-debugger/client-debugger-view/src/Debugger.tsx

```diff
@@ -32,48 +39,68 @@ export interface FluidClientDebuggerProps {
  * @remarks If no debugger has been initialized, will display a note to the user and a refresh button to search again.
  */
 export function FluidClientDebugger(props: FluidClientDebuggerProps): React.ReactElement {
-    /**
-     * This function will retrieve all client debuggers and return the first one.
-     * TODO: update it to check container id after support multi-containers app.
-     */
     function getFirstDebugger(): IFluidClientDebugger | undefined {
-        const clientDebuggers = getFluidClientDebuggers();
-        return clientDebuggers.length === 0 ? undefined : clientDebuggers[0];
+        const debuggers = getFluidClientDebuggers();
+        return debuggers.length === 0 ? undefined : debuggers[0];
     }
 
+    const [clientDebuggers, setClientDebuggers] = React.useState<IFluidClientDebugger[]>(getFluidClientDebuggers());
+
+    const [selectedContainerId, setSelectedContainerId] = React.useState<string>(
```

Comment:
Instead of using an empty string as the default value here, I would recommend allowing this value to be `undefined` and having its default be `undefined`.
```suggestion
    const [selectedContainerId, setSelectedContainerId] = React.useState<string | undefined>(
```

Reply:
Fixed